lib/elix/responders/guys.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Elix.Responders.Guys do
+  @moduledoc """
+  Encourages the use of gender-neutral forms of address
+  """
+
+  use Hedwig.Responder
+
+  @forms_of_address [
+    "all y'all",
```

Comment:
I'd love to use curly quotes here, e.g. `’` (`Opt+Shift+]`) instead of straight ones. Also on line 19.